client/packages/invoices/src/Returns/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -0,0 +1,195 @@
+  currentStatus: InvoiceNodeStatus,
+  getButtonLabel: (status: InvoiceNodeStatus) => string
+): SplitButtonOption<InvoiceNodeStatus>[] => {
+  const options: [
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+    SplitButtonOption<InvoiceNodeStatus>,
+  ] = [
+    {
+      value: InvoiceNodeStatus.New,
+      label: getButtonLabel(InvoiceNodeStatus.New),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Allocated,
+      label: getButtonLabel(InvoiceNodeStatus.Allocated),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Picked,
+      label: getButtonLabel(InvoiceNodeStatus.Picked),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Shipped,
+      label: getButtonLabel(InvoiceNodeStatus.Shipped),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Delivered,
+      label: getButtonLabel(InvoiceNodeStatus.Delivered),
+      isDisabled: true,
+    },
+    {
+      value: InvoiceNodeStatus.Verified,
+      label: getButtonLabel(InvoiceNodeStatus.Verified),
+      isDisabled: true,
+    },
+  ];
+
+  if (currentStatus === InvoiceNodeStatus.New) {
+    options[1].isDisabled = false;
+    options[2].isDisabled = false;
+    options[3].isDisabled = false;
+  }
+
+  if (currentStatus === InvoiceNodeStatus.Allocated) {
+    options[2].isDisabled = false;
+    options[3].isDisabled = false;
+  }
+
+  if (currentStatus === InvoiceNodeStatus.Picked) {
+    options[3].isDisabled = false;
+  }
+
+  return options;
+};
+
+const getNextStatusOption = (
+  status: InvoiceNodeStatus,
+  options: SplitButtonOption<InvoiceNodeStatus>[]
+): SplitButtonOption<InvoiceNodeStatus> | null => {
+  if (!status) return options[0] ?? null;
+
+  const nextStatus = getNextOutboundStatus(status);
+  const nextStatusOption = options.find(o => o.value === nextStatus);
+  return nextStatusOption || null;
+};
+
+const getButtonLabel =
+  (t: ReturnType<typeof useTranslation>) =>
+  (invoiceStatus: InvoiceNodeStatus): string => {
+    return t('button.save-and-confirm-status', {
+      status: t(getStatusTranslation(invoiceStatus)),
+    });
+  };
+
+const useStatusChangeButton = () => {
+  // const { lines, status, onHold, update } = useOutbound.document.fields([
+  //   'status',
+  //   'onHold',
+  //   'lines',
+  // ]);
+
+  const { success, error } = useNotification();
+  const t = useTranslation('distribution');
+  const { data } = useReturns.document.outboundReturn();
+
+  // TEMP until "fields" hook available:
+  const status = data?.status ?? InvoiceNodeStatus.New;
+  const update = async (_: unknown) => true;
+  const onHold = false;
+  const lines: any = [];
+
+  const hasLinesToPrune =
+    data?.status === InvoiceNodeStatus.New &&
+    (data?.lines?.nodes ?? []).some(line => line.numberOfPacks === 0);
```

Comment:
Shouldn't be possible for returns, we require a number of packs to be set, or we remove the line on the backend